doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,216 @@
+approach follows the trajectory of a possible pathway step by step starting at a
+seed; however, unlike deterministic tracking, the tracking direction at each
+point along the path is chosen at random from a distribution. The distribution
+at each point is different and depends on the observed diffusion data at that
+point. The distribution of tracking directions at each point can be represented
+as a probability mass function (PMF) if the possible tracking directions are
+restricted to discrete numbers of well distributed points on a sphere.
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the probabilistic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz data/stanford_hardi/pve_wm.nii.gz --seed_density 2 --tracking_method "prob" --out_dir "det_tracking_output"
+
+Closest Peaks Tracking
+**********************
+
+Closest peaks fiber tracking is a type of deterministic tracking. Deterministic
+streamlines follow a predictable path through the data by selecting the same
+diffusion orientation when evaluating the same point through the propagation
+process. There may be several estimated directions at each point (such as a
+voxel center): closest peaks tracking selects one of these estimated directions
+on the basis of closeness of match to the previous direction of the streamline.
+
+We will first create the directory in which to save the output tractogram
+file (e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the closest peaks tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz data/stanford_hardi/pve_wm.nii.gz --seed_density 2 --tracking_method "cp" --out_dir "cp_tracking_output"
+
+---------------------------------
+Particle Filtering Tracking (PFT)
+---------------------------------
+
+Particle Filtering Tracking (PFT) [Girard2014]_ uses tissue partial volume
+estimation (PVE) to reconstruct trajectories connecting the gray matter, and
+not prematurely stopping in the white matter or in the corticospinal fluid. It
+relies on a stopping criterion that identifies the tissue where the streamline
+stopped. If the streamline reaches the gray matter, the trajectory is kept. If
+the streamline incorrectly stopped in the white matter or in the cerebrospinal
+fluid, PFT uses anatomical information to find an alternative streamline
+segment to extend the trajectory. When this segment is found, the tractography
+continues until the streamline reaches the gray matter.
+
+We will use the ``stanford_hardi`` dataset in DIPY to showcase this tracking
+method. As with any other workflow in DIPY, you can also use your own data!
+
+We will first create a directory in which to save the output tractogram file
+(e.g.: ``pf_track_output``)::
```

Comment:
For the sake of consistency, can the example output directory be renamed to {double_backticks}pft_output{double_backticks}, please? Apply that change to [here](https://github.com/dipy/dipy/pull/2292/files#diff-c3f6f4adec7e27501363531d0782fec37f9c10e7885ffd9de9aea11badb43602R175) as well.

Reply:
Addressed in bbb36f1